cmd/cstor-volume-mgmt/controller/volume-controller/handler.go

```diff
@@ -57,23 +57,27 @@ func (c *CStorVolumeController) syncHandler(key string, operation common.QueueOp
 
 		_, err := c.clientset.OpenebsV1alpha1().CStorVolumes(cStorVolumeGot.Namespace).Update(cStorVolumeGot)
 		if err != nil {
+			glog.Infof("failed to update cStorVolume:%v, %v; Status: %v err: %v", cStorVolumeGot.Name,
+				string(cStorVolumeGot.GetUID()), cStorVolumeGot.Status.Phase, err)
 			return err
```

Comment:
Will it be good to wrap the err with the info we are logging?
Should we log this as Infof?